GITFLOW.md

```diff
@@ -0,0 +1,58 @@
+# Git Flow
+The team uses Git Flow to to develop the application and it's API.
```

Comment:
Maybe also worth linking here to an article/explainer for someone who wants to find out a bit more?

Reply:
I'll link to the [Atlassian documentation](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) and the [cheat sheet](https://danielkummer.github.io/git-flow-cheatsheet/index.fr_FR.html)?

Reply:
Sounds good - thanks!